ena-submission/scripts/call_loculus.py

```diff
@@ -115,11 +119,14 @@ def submit_external_metadata(metadata_file, config: Config, organism: str) -> re
         "Content-Type": "application/x-ndjson",
     }
 
-    with open(metadata_file, encoding="utf-8") as file:
-        pre_ndjson = [x.strip() for x in file]
-    data = " ".join(pre_ndjson)  # Q: Space not newline?
+    data = json.dumps(external_metadata)
+
+    response = make_request(HTTPMethod.POST, url, config, data=data, headers=headers, params=params)
 
-    return make_request(HTTPMethod.POST, url, config, data=data, headers=headers, params=params)
+    if not response.ok:
+        response.raise_for_status()
+
+    return response
```

Comment:
Huh? Why this change? I thought that `make_request` will already raise an error if the response is not ok.